dags/openshift_nightlies/dag.py

```diff
@@ -194,6 +196,43 @@ def build(self):
     def _get_openshift_installer(self):
         return rogcp.RoGCPInstaller(self.dag, self.config, self.release)
 
+class PrebuiltOpenshiftNightlyDAG(AbstractOpenshiftNightlyDAG):
+    def __init__(self,openshift_release, DagConfig):
```

Comment:
DagConfig is a class name but should be a param, i think you were meaning to type hint like `dag_config: DagConfig`

Reply:
right